packages/cli/tests/create-tutorial.test.ts

```diff
@@ -69,6 +69,66 @@ test('create and build a project', async (context) => {
   expect(distFiles.map(normaliseSlash).sort()).toMatchSnapshot();
 });
 
+test('create and eject a project', async (context) => {
```

Comment:
Maybe also try to eject from something that is not an TK project?

Reply:
Ah that's a very good point! I should write a test for that too :+1:

Reply:
Tests added :+1: